src/wp-includes/blocks/legacy-widget.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Server-side rendering of the `core/legacy-widget` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the 'core/legacy-widget' block.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Rendered block.
+ */
+function render_block_core_legacy_widget( $attributes ) {
+	if ( isset( $attributes['id'] ) ) {
+		$sidebar_id = gutenberg_find_widgets_sidebar( $attributes['id'] );
```

Comment:
Can we remove `gutenberg_` form all these functions?

Reply:
Yes this was done in https://github.com/WordPress/gutenberg/pull/32176. But that PR had to be merged after this one, hence the `gutenberg_` compatibility functions which I'll remove shortly 😀